changelogs/fragments/502-zfs_bugfix_and_diff_mode_support.yaml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - zfs.py - treated received properties as local
```

```suggestion
  - zfs - treated received properties as local
```
The module is called `zfs`, not `zfs.py`. (Same below.)

---

I changed the changelog fragment in order to reflect your suggestions and commited it to my branch [froebela/community.general:patch-1](https://github.com/froebela/community.general/tree/patch-1).